config/config.yaml

```
@@ -25,6 +25,21 @@ ref:
   release: 111
   # Genome build
   build: GRCh38
+  pangenome: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)